src/kube-enricher.go

```diff
@@ -72,21 +75,31 @@ func main() {
 		informers.DebugInfo(log.Writer())
 	}
 
-	scanner := bufio.NewScanner(os.Stdin)
-	for scanner.Scan() {
-		in := scanner.Bytes()
-		enriched, err := enrich(informers, in, mapping)
+	var format format.Format = nil
```

Comment:
if i'm correct `= nil` is not necessary here, interfaces are nil by default (zero-value)